developer-docs-site/docs/guides/getting-started.md

```diff
@@ -26,45 +26,52 @@ Start by cloning the `aptos-core` GitHub repo from [GitHub](https://github.com/a
     cd aptos-core
     ```
 
-3. Run the `scripts/dev_setup.sh` Bash script as shown below. This will prepare your developer environment.
+3. Run the `scripts/dev_setup.sh` Bash script as shown below. This will prepare your developer environment by installing all dependencies needed to build, test and inspect Aptos Core. Note, you may be prompted for your password:
 
     ```
     ./scripts/dev_setup.sh
     ```
 
-4. Update your current shell environment.
+TODO: ID which OSes this script works on. macOS and Linux only? And ask if we can remove the selection verbiage in the output of “Based on your selection, these tools will be included”
```

Comment:
Alert to remove TODOs from here into the Linear issue.

Reply:
I have moved all of the TODOs to the Linear issue and removed them from here.